elasticsearch/probe/readiness.sh

```diff
@@ -40,4 +40,11 @@ function check_if_ready() {
   fi
 }
 
+function check_for_init_complete() {
+  if [ ! -f ${HOME}/init_complete ]; then
```

Comment:
you could change this function to be just one line:
```
test -f ${HOME}/init_complete
```
test will return 0 (success) if the file exists and non-zero otherwise
the default return value of a function is the last code returned by a command in that function